101-container-registry/azuredeploy.parameters.json

```diff
@@ -2,14 +2,11 @@
   "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
   "contentVersion": "1.0.0.0",
   "parameters": {
-    "acrName": {
+    "registryName": {
       "value": "GEN-UNIQUE"
     },
-    "acrAdminUserEnabled": {
+    "adminUserEnabled": {
```

Comment:
NIT: can be omitted since it defaults to `false`

Reply:
Yes.. the default is false but we allow a user to set it when clicking the deploy to azure button.

Reply:
do we need to include it, suggesting it can/should be changed? Or, can we focus on the default scenario to use per-user identity, with service principals today?

Reply:
Should update this when we enable token based flow. Basically this is the non-aad route so leaving as it is for now.